Wikipedia/Code/TalkPageFormattingToolbarView.swift

```diff
@@ -0,0 +1,155 @@
+        return stackView
+    }()
+
+    lazy var italicsButtonSeparatorView: UIView =  {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .gray
+        return view
+    }()
+
+    lazy var italicsButtonStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.distribution = .fillProportionally
+        stackView.axis = .horizontal
+        return stackView
+    }()
+
+    lazy var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.distribution = .fillEqually
+        stackView.axis = .horizontal
+        return stackView
+    }()
+
+    init() {
+        super.init(frame: CGRect.init(x: 0, y: 0, width: UIScreen.main.bounds.width, height: 44))
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func setup() {
+        addSubview(stackView)
+
+        stackView.addArrangedSubview(boldButtonStackView)
+        stackView.addArrangedSubview(italicsButtonStackView)
+        boldButtonStackView.addArrangedSubview(boldButton)
+        boldButtonStackView.addArrangedSubview(boldButtonSeparatorView)
+        italicsButtonStackView.addArrangedSubview(italicsButton)
+        italicsButtonStackView.addArrangedSubview(italicsButtonSeparatorView)
+
+        stackView.addArrangedSubview(linkButton)
+
+        NSLayoutConstraint.activate([
+            stackView.topAnchor.constraint(equalTo: topAnchor),
+            stackView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            stackView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            stackView.bottomAnchor.constraint(equalTo: bottomAnchor),
+
+            boldButton.centerYAnchor.constraint(equalTo: stackView.centerYAnchor),
+            italicsButton.centerYAnchor.constraint(equalTo: stackView.centerYAnchor),
+
+            boldButtonSeparatorView.heightAnchor.constraint(equalTo: boldButton.heightAnchor, constant: -20),
+            boldButtonSeparatorView.widthAnchor.constraint(equalToConstant: 0.5),
+            boldButtonSeparatorView.centerYAnchor.constraint(equalTo: stackView.centerYAnchor),
+
+            italicsButtonSeparatorView.heightAnchor.constraint(equalTo: italicsButton.heightAnchor, constant: -20),
+            italicsButtonSeparatorView.widthAnchor.constraint(equalToConstant: 0.5),
+            italicsButtonSeparatorView.centerYAnchor.constraint(equalTo: stackView.centerYAnchor)
+        ])
+        configure()
+
+    }
+
+    @objc func didTapBoldButton() {
+        delegate?.didSelectBold()
+    }
+
+    @objc func didTapItalicsButton() {
+        delegate?.didSelectItalics()
+    }
+
+    @objc func didTapInsertLinkButton() {
+        delegate?.didSelectInsertLink()
+    }
+
+    func configure() {
```

Comment:
I would add private around most of these properties and methods in here if you don't intend to call them from the outside.